tests/ui/consts/min_const_fn/mutable_borrow.rs

```diff
@@ -1,24 +0,0 @@
-<<<PULL-UPSTREAM>>> file deleted upstream; move the Ferrocene annotations if any, and delete this file
-const fn mutable_ref_in_const() -> u8 {
-    let mut a = 0;
-    let b = &mut a; //~ ERROR mutable references
-    *b
-}
-
-struct X;
-
-impl X {
-    const fn inherent_mutable_ref_in_const() -> u8 {
-        let mut a = 0;
-        let b = &mut a; //~ ERROR mutable references
-        *b
-    }
-}
-
-fn main() {}
-
-// ferrocene-annotations: fls_qztk0bkju9u
```

Comment:
Do these annotations need to be moved?

Reply:
I have no idea o.O. I just cherry-picked this from https://github.com/ferrocene/ferrocene/pull/947

Reply:
If the test got deleted and we have other tests with the annotations they can go, if the test is deleted and we don't have others we need to find new tests for this.

If the test got moved the annotation ought to move as well.

Reply:
We have other 88 tests with the same annotation `fls_qztk0bkju9u`

Reply:
Thats fine then